source/Patches/GameSettings.cs

```diff
@@ -110,7 +110,9 @@ public static class Update
         {
             public static void Postfix(ref GameOptionsMenu __instance)
             {
-                __instance.GetComponentInParent<Scroller>().YBounds.max = 84;
+                /*Automatically calculate the Ybounds.max
+                    for the 2021.11.9 the value was (__instance.Children.Length - 8.5f) / 2;*/
+                __instance.GetComponentInParent<Scroller>().YBounds.max = (__instance.Children.Length - 7) / 2 + 0.2f;
```

Comment:
Would be very cool if you could document where these numbers came from so it's not just magic numbers.

Reply:
It's very simple, each option is equal to 0.5, 2 options are equal to 1.
the first 7 options are not counted because they are visible without scroll.
the margin at the very end is equal to 0.2
we therefore take __instance.children.length which corresponds to the number of total options,
we subtract 7 and divide by 2 and we finish by adding 0.2 for the final margin

Reply:
Can you put that into a comment in the actual code?

Reply:
yes